applications/openshift/general/resource_requests_limits_in_deployment/rule.yml

```diff
@@ -12,22 +12,24 @@ description: |-
   for a container to use more resource than its request for that resource specifies.
   However, a container is not allowed to use more than its resource limit.
 
-# todo : better if we can keep openshift document link for this reference,
+  <p>
+  To configure resource requests/limits for a deployement, follow the directions in
+  {{{ weblink(link="https://docs.openshift.com/container-platform/latest/applications/deployments/managing-deployment-processes.html#deployments-setting-resources_deployment-operations",
```

Comment:
This will redirect to https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/building_applications/deployments#deployments-setting-resources_deployment-operations now - we could update it here to use docs.redhat.com so we don't have to later.